prboom2/src/st_stuff.c

```diff
@@ -994,11 +991,6 @@ static void ST_loadData(void)
   ST_loadGraphics(true);
 }
 
-static void ST_unloadData(void)
```

Comment:
Data never gets unloaded...

Reply:
If you're removing this, please update ST_loadGraphics to not take an argument (indeed, looks like that argument isn't used anyway) and unroll the ST_loadData function as well.

Reply:
Was data supposed to get unloaded, but it was never implemented?  Does this leak memory?

Reply:
It doesn't look like this has ever done what it claims.  Removed.